src/js/videojs-dash.js

```diff
@@ -89,6 +91,12 @@ class Html5DashJS {
     this.mediaPlayer_.setProtectionData(this.keySystemOptions_);
     this.mediaPlayer_.attachSource(manifestSource);
 
+    // Setup text tracks
+    setupTextTracks.call(null, this.player, tech);
```

Comment:
My concern is that `dashjs.MediaPlayer.events.TEXT_TRACKS_ADDED`/`dashjs.MediaPlayer.events.PLAYBACK_METADATA_LOADED` may fire between `attachSource()` and the event listener setup functions